src/scos_tekrsa/configs/actions-500-600/CBRS_EXT_DataProduct_08dB_Attenuation.yml

```diff
@@ -11,18 +11,55 @@ nasctn_sea_data_product:
 # PFP frame
   pfp_frame_period_ms: 10
 # APD downsampling settings
-  apd_bin_size_dB: 0.5  # Set to 0 or negative for no downsampling
+  apd_bin_size_dB: 1.0  # Set to 0 or negative for no downsampling
+  apd_max_bin_dBm: -30
+  apd_min_bin_dBm: -180
 # Time domain power statistics settings
   td_bin_size_ms: 10
 # Sigan Settings
   preamp_enable: True
-  reference_level: -25
-  attenuation: 15
+  reference_level:
+    - -25
+    - -25
+    - -25
+    - -25
+    - -25
+    - -25
+    - -25
+    - -25
+    - -25
+    - -25
+    - -25
+    - -17
+    - -17
+    - -17
+    - -17
+    - -17
+    - -17
+  attenuation:
+    - 0
+    - 0
+    - 0
+    - 0
+    - 0
+    - 0
+    - 0
+    - 0
+    - 0
+    - 0
+    - 0
+    - 8
+    - 8
+    - 8
+    - 8
+    - 8
+    - 8
   sample_rate: 14e6
-# Acquisition settings (3550-3700 MHz in 10 MHz steps, each 4s long)
+# Acquisition settings (3540-3700 MHz in 10 MHz steps, each 4s long)
```

Comment:
It looks like this comment here and in other yml files should be updated to be 3540-3710 MHz

Reply:
Addressed in 0994e63f3ea3f57789c7073e1f401afb18311048